src/main/java/net/atos/zac/app/ontkoppeldedocumenten/OntkoppeldeDocumentenRESTService.java

```diff
@@ -92,8 +91,18 @@ public RESTResultaat<RESTOntkoppeldDocument> list(final RESTOntkoppeldDocumentLi
         assertPolicy(policyService.readWerklijstRechten().inbox());
         final OntkoppeldDocumentListParameters listParameters = listParametersConverter.convert(restListParameters);
         final OntkoppeldeDocumentenResultaat resultaat = ontkoppeldeDocumentenService.getResultaat(listParameters);
+        var ontkoppeldeDocumenten = resultaat.getItems();
+        var informationObjectTypeUUIDs = ontkoppeldeDocumenten.stream().map(
+                ontkoppeldeDocument -> extractUuid(
+                        drcClientService
+                                .readEnkelvoudigInformatieobject(ontkoppeldeDocument.getDocumentUUID())
+                                .getInformatieobjecttype()
+                )
+        ).toList();
```

Comment:
Is this toList() needed? In Kotlin at least not because the result of .map() is already a list.

Reply:
Not feasible to convert this to Kotlin in this PR I guess? ;-)

Reply:
`toList()` is needed. I would prefer to have the conversion in a separate PR. Too many changes already in this one.